sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -413,7 +413,7 @@ abstract class JdbcDialect extends Serializable with Logging {
 
     override def visitSQLFunction(funcName: String, inputs: Array[String]): String = {
```

Comment:
why not follow the existing API style and add an overload that takes inputs as v2 Expression instead of string?

Reply:
I want the visitSQLFunction of JDBCSQLBuilder could support the unified check for supported functions.
```
if (isSupportedFunction(funcName)) {
  // compiled DIY
} else {
  throw new SparkUnsupportedOperationException.
}
```